resources.go

```diff
@@ -47,6 +47,7 @@ const (
 	azureAD                  = "AD"                  // Active Directory (AAD)
 	azureAnalysisServices    = "AnalysisServices"    // Analysis Services
 	azureAPIManagement       = "ApiManagement"       // API Management
+	azureAppConfiguration    = "AppConfiguration"    // AppConfiguration
```

Comment:
Nit to match the product name
```suggestion
	azureAppConfiguration    = "AppConfiguration"    // App Configuration
```